cmd/dump.go

```diff
@@ -215,6 +215,12 @@ func fatal(format string, args ...interface{}) {
 }
 
 func runDump(ctx *cli.Context) error {
+	var file *os.File
+	fileName := ctx.String("file")
+	if fileName == "-" {
+		file = os.Stdout
+		log.DelLogger("console")
```

Comment:
Haven't found a better way to setup the console logger to log to stderr.
So I just deleted the logger as logs may break the file written to stdout.